pkg/registry/resource/resourceclaimspec/strategy.go

```diff
@@ -0,0 +1,177 @@
+/*
+Copyright 2022 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package resourceclaimspec
```

Comment:
It looks a little strange to have a registry directory for something that isn't a top-level resource. It looks like every other instance of an exported `DropDisabled.*Fields` function is under `pkg/api/.../util.go` (e.g. `pkg/api/resourceclaim(spec)/util.go`)